update steps for cluster page changes (#11883)
pingcap/docs: tidb-cloud/set-up-private-endpoint-connections.md

To start setting up a private endpoint, open the private endpoint creation page:

1. Log in to the [TiDB Cloud console](https://tidbcloud.com).

> 2. In the left navigation pane, do one of the following:

Author: ```suggestion
2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:
```

>     - If you have multiple projects, choose a target project, and then click **Admin** > **Network Access**.

Author: ```suggestion
    - If you have multiple projects, switch to the target project, and then click **Admin** > **Network Access**.
```
